test/index.spec.js

```diff
@@ -65,24 +63,28 @@ var baseReporterDecorator = function(context) {
   context.writeCommonMsg = sinon.spy();
   context.write = sinon.spy();
 };
-
+var originalPlatform = '';
 describe('SpecReporter', function() {
   describe('when initializing', function() {
     describe('and on a windows machine', function() {
+      after(function(done){
+        Object.defineProperty(process, 'platform', {
+           value: originalPlatform
+        });
+        done();
+      })
       function createSpecReporter(config) {
         config = config || {};
         var processMock = {
           platform: function() {
             return 'win32';
           }
         };
```

Comment:
`processMock` doesn't look used; can it be deleted?

Reply:
good catch, thanks!